.multi-releaserc

```diff
@@ -0,0 +1,4 @@
+{
+    "ignorePrivate": false,
+    "ignorePackages": ["!packages/**"]
```

Comment:
Why are private packages not ignored?

Reply:
To quote me from [here](https://github.com/ReVanced/revanced-helper/pull/41#discussion_r1676234938):

> `ignorePrivate: false`: Our packages/projects like bots, and API doesn't need to be published on any registry, so we set `"private": true` in their `package.json`s, but we still need to build them.